tools/make_release.py

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/python
```

Comment:
If this is a python 3 script, `python` usually defaults to `/usr/bin/python2` on most systems, when run using `./make_release.py`.